### PR TITLE
Added DRT Strategies Inc GRC tool 

### PR DESCRIPTION
**Pull request by vmangat**
# Committer Notes

{Please provide a brief description of what this PR accomplishes. Be sure to reference any issues addressed. If the PR is a work-in-progress submitted for early review, please include [WIP] at the beginning of the title or mark the PR as DRAFT.}

### All Submissions:

- [x] Have you selected the correct base branch per  [Contributing](https://github.com/usnistgov/OSCAL/blob/master/CONTRIBUTING.md) guidance?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [X] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.
- ~Have you added an explanation of what your changes do and why you'd like us to include them?~
- ~Have you written new tests for your core changes, as applicable?~
- ~Have you included examples of how to use your new feature(s)?~

### CODE REVIEW

File: docs/content/tools/_index.md [hunk 1]
```diff
@@ -39,3 +39,4 @@ See the [NIST Software Disclaimer](https://www.nist.gov/disclaimer) for more inf
...
 | [control_freak](https://controlfreak.risk-redux.io/) | Risk Redux | This tool seeks to provide folks with a searchable and easy-to-navigate reference for NIST SP 800-53 Revision 5. It is [an open-source application from the Risk Redux project](https://github.com/risk-redux/control_freak), built using parsed content directly from the OSCAL repositories. | open-source |
+| [DRT Confidence](https://www.drtstrategies.com/drt-confidence-fedramp/) | DRT Strategies Inc. | DRTConfidence is a next generation Governance, Risk and Compliance (GRC) solution to help organizations transition to OSCAL and continuous compliance. DRTConfidence is available in a FedRAMP High environment, supports ALL OSCAL artifacts: Catalogs, Profiles, System Security Plans, Security Assessment Plans, Security Assessment Report, POAM and is compliant with OSCAL v1.0.0. Additional Information can be found at [DRT Confidence for FedRAMP](https://www.drtstrategies.com/drt-confidence-fedramp/). | Commercial License |
```
Comment (aj-stein-nist) [inline docs/content/tools/_index.md:42]: Very minor grammar nits around plurals and phrasing (there is no one true FedRAMP High Environment, but many possible environments, right?) and some other minor stuff.

Thanks for bringing it back up to HEAD and getting rid of the need for rebasing! :-)

```suggestion
| [DRT Confidence](https://www.drtstrategies.com/drt-confidence-fedramp/) | DRT Strategies Inc. | DRTConfidence is a next generation Governance, Risk and Compliance (GRC) solution to help organizations transition to OSCAL and continuous compliance. DRTConfidence is available in FedRAMP High environments and supports all OSCAL artifacts: Catalogs, Profiles, System Security Plans, Security Assessment Plans, Security Assessment Reports, POAMs and conforms to the OSCAL v1.0.0 specification and its schemas. Additional information can be found at [DRT Confidence for FedRAMP](https://www.drtstrategies.com/drt-confidence-fedramp/). | Commercial License |
```

**Review by aj-stein-nist — CHANGES_REQUESTED** [review]
Some minor issues, but nice work!

File: docs/content/tools/_index.md [hunk 2]
```diff
@@ -39,3 +39,4 @@ See the [NIST Software Disclaimer](https://www.nist.gov/disclaimer) for more inf
...
 | [control_freak](https://controlfreak.risk-redux.io/) | Risk Redux | This tool seeks to provide folks with a searchable and easy-to-navigate reference for NIST SP 800-53 Revision 5. It is [an open-source application from the Risk Redux project](https://github.com/risk-redux/control_freak), built using parsed content directly from the OSCAL repositories. | open-source |
+| [DRT Confidence](https://www.drtstrategies.com/drt-confidence-fedramp/) | DRT Strategies Inc. | DRTConfidence is a next generation Governance, Risk and Compliance (GRC) solution to help organizations transition to OSCAL and continuous compliance. DRTConfidence is available in FedRAMP High environments and supports all OSCAL artifacts: Catalogs, Profiles, System Security Plans, Security Assessment Plans, Security Assessment Reports, POAMs and conforms to the OSCAL v1.0.0 specification and its schemas. Additional information can be found at [DRT Confidence for FedRAMP](https://www.drtstrategies.com/drt-confidence-fedramp/). | Commercial License |
```
Comment (iMichaela) [inline docs/content/tools/_index.md:42]: Is the Component Definition not supported or omitted by mistake? If not supported, maybe the statement should not read 'all OSCAL artifacts'. If omitted by mistake, please include it.

Comment (vmangat) [reply docs/content/tools/_index.md]: It was omitted by mistake, we will add it and resubmit

**Comment by vmangat** [comment]
AJ and Michaela, thank you for your feedback.  I have committed the edits.

- valinder

**Review by aj-stein-nist — APPROVED** [review]
Updates look good on my end. Thanks for quickly our addressing our feedback.

**Pull request merged**